Description: Another vga arbiter fix for supporting more than one GPU with kernel 2.6.32
Origin: upstream, http://www.nvnews.net/vbulletin/attachment.php?attachmentid=40165&d=1272937909
Bug: http://www.nvnews.net/vbulletin/showthread.php?p=2243704
Bug-Debian: http://bugs.debian.org/566874
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/nvidia-graphics-drivers/+bug/548362
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/nvidia-graphics-drivers/+bug/573557

diff -ru usr/src/nv/nv.c usr/src/nv.682377/nv.c
--- a/nv.c	2010-04-22 11:49:41.000000000 -0700
+++ b/nv.c	2010-05-03 15:24:45.330672559 -0700
@@ -4749,6 +4749,9 @@
     pci_set_master(dev);
 
 #if defined(CONFIG_VGA_ARB)
+#if defined(VGA_DEFAULT_DEVICE)
+    vga_tryget(VGA_DEFAULT_DEVICE, VGA_RSRC_LEGACY_MASK);
+#endif
     vga_set_legacy_decoding(dev, VGA_RSRC_NONE);
 #endif
 
